### PR TITLE
Refactored two midi send methods into one with timeStamp defaulted to mach_absolute_time()

### PR DESCRIPTION
**Pull request by Matt54**
Refactored two midi send methods into one with timeStamp defaulted to mach_absolute_time()

This is a solution to: #2676

### CODE REVIEW

File: Sources/AudioKit/MIDI/MIDI+Sending.swift [hunk 1]
```diff
@@ -352,27 +287,31 @@ extension MIDI {
...
         let message: [MIDIByte] = [noteCommand, noteNumber, velocity]
-        self.sendMessage(message, endpointsUIDs: endpointsUIDs, virtualOutputPorts: virtualOutputPorts)
+        self.sendMessage(message, time: time, endpointsUIDs: endpointsUIDs, virtualOutputPorts: virtualOutputPorts)
     }
 
     /// Send a Note Off Message
     /// - Parameters:
     ///   - noteNumber: MIDI Note Number
     ///   - channel: MIDI Channel (default: 0)
+    ///   - time: MIDI Timestamp (default: mach_absolute_time())
```
Comment (orchetect) [inline Sources/AudioKit/MIDI/MIDI+Sending.swift:306]: May want to add a note to the inline doc here that `time` should never be a value of 0

File: Sources/AudioKit/MIDI/MIDI+Sending.swift [hunk 2]
```diff
@@ -409,50 +348,14 @@ extension MIDI {
...
-
     /// Send Message with data with timestamp
     /// - Parameters:
     ///   - data: Array of MIDI Bytes
-    ///   - time: MIDI Timestamp
-    public func sendMessageWithTime(_ data: [MIDIByte],
-                                    time: MIDITimeStamp,
-                                    endpointsUIDs: [MIDIUniqueID]? = nil,
-                                    virtualOutputPorts: [MIDIPortRef]? = nil) {
+    ///   - time: MIDI Timestamp (default mach_absolute_time())
```
Comment (orchetect) [inline Sources/AudioKit/MIDI/MIDI+Sending.swift:354]: And also here - see other note about inline doc

**Comment by orchetect** [comment]
`MIDITimeStamp` is a typealias of UInt64, so constructing it is redundant.

`MIDITimeStamp(mach_absolute_time())` can just be `mach_absolute_time()`

**Comment by Matt54** [comment]
Thanks for the comments @orchetect! Committed the changes

**Comment by orchetect** [comment]
Thanks for the PR!

**Pull request merged**